docs/data/sdks/java/index.md

```diff
@@ -198,17 +198,39 @@ client.logEvent(event);
 
 #### Events with groups
 
+--8<-- "includes/groups-intro-paragraph.md"
+
+!!! example
+    If Joe is in 'orgId' '10', then the `groupName` would be '10':
```

Comment:
Add the example of using array as a groupName will be better.

Reply:
Added both examples here